opencti-platform/opencti-front/src/components/ItemEntityType.tsx

```diff
@@ -28,28 +33,92 @@ const useStyles = makeStyles(() => ({
 
 interface ItemEntityTypeProps {
   entityType: string;
-  variant?: string;
+  maxLength?: number;
+  inList?: boolean;
+  showIcon?: boolean;
+  isRestricted?: boolean;
+  style?: React.CSSProperties;
+  size?: 'small' | 'medium' | 'large'
 }
 
 const ItemEntityType: FunctionComponent<ItemEntityTypeProps> = ({
-  variant = 'inList',
+  inList = true,
+  maxLength,
   entityType,
+  showIcon = false,
+  isRestricted = false,
+  style = {},
+  size,
```

Comment:
```suggestion
  size = 'medium',
```
to facilitate code comprehension (otherwise you have to dig into the code, see the switch)